plugins/networktables/src/main/java/edu/wpi/first/shuffleboard/plugin/networktables/sources/SingleKeyNetworkTableSource.java

```diff
@@ -44,18 +45,41 @@ public SingleKeyNetworkTableSource(ITable table, String key, DataType dataType)
         // The change was from network tables; setting the value again would be redundant
         return;
       }
-      if (table.getValue(key, null) == newValue || !isConnected()) {
+      if (table.getEntry(key).getValue() == newValue || !isConnected()) {
```

Comment:
It's more efficient to save the return value of table.getEntry(key) and then use it in the lambda.